src/tests/performance/performance.py

```diff
@@ -0,0 +1,96 @@
+import configparser
+import subprocess
+import datetime
+import json
+import sys
+import os
+import re
+
+from pathlib import Path
+
+# needed to find the execution_info.ini
+def get_latest_output_folder(path):
+    output_path_study = os.path.join(path, "output")
+    files = os.listdir(output_path_study)
+    paths = [os.path.join(output_path_study, basename) for basename in files]
+    return max(paths, key=os.path.getctime)
+
+def search_patern_in_file(file_name, pattern):
+    f = open(file_name)
+    s = f.read()
+    f.close()
```

Comment:
`with open(file_name) as f:`

https://docs.python.org/fr/3/reference/compound_stmts.html#the-with-statement